NativeApp/src/components/User/View/styles.js

```diff
@@ -34,15 +34,12 @@ const styles = StyleSheet.create({
     marginHorizontal: 20,
   },
   profileName: {
-    paddingBottom: 10,
-    paddingTop: 10,
     alignSelf: 'center',
-    flexDirection: 'row',
   },
   flatListView: {
     flex: 1,
   },
-  H3Bold: {
+  profileText: {
     fontWeight: 'bold',
```

Comment:
Noticing a lot of bold font type being added to Texts. Maybe better add a bold type to the Texts common components?

Reply:
Yes, i agree. I have added the bold type to the component.